engine/node_latex.cc

```diff
@@ -374,6 +374,18 @@ namespace MathDAG
   {
     return o<<" t ";
   }
+  
+  template <>
+  ostream& OperationDAG<OperationType::euler>::latex(ostream& o) const
+  {
+    return o<<" e ";
+  }
+ 
+  template <>
+  ostream& OperationDAG<OperationType::pi>::latex(ostream& o) const
+  {
+    return o<<" π ";
```

Comment:
LaTeX for pi is \pi.

Reply:
Thanks for catching this. My goodness, all the articles I've published were written in Latex. It does not cease to amaze me how being in a different context can render acquired knowledge/skills almost useless...